### PR TITLE
chore(docs): update readme

### PR DESCRIPTION
**Pull request by fokusferit**
**What kind of change does this PR introduce?**

We did so much work in the past, that our Docs (Readme, etc.) are a bit out of sync. With this PR I#m starting to getting all pieces together step by step. Starting with the most important entry point, the README.

**Did you add tests for your changes?**
not necessary
**If relevant, did you update the documentation?**
Yes.
**Summary**
Updating Readme to reflect our new structure. It is highly inspired by the Lerna Readme. It is part of #247 . 

Our README reflects now all the existing cli commands, references the docs for them and describes shortly our approach.

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,25 +19,56 @@
...
+When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed! Otherwise you would need to install webpack CLI and the packages you want to use. If we want to use the `init` functionality to create a new `webpack.config.js` configuration file:
+
+```sh
+npm i webpack-cli @webpack-cli/init
+npx webpack-cli init
+```
+
+After asking multiple questions related to your project and as a result create a webpack.config.js based on your selected answers. 
+
+## How it works
```
Comment (rishabh3112) [inline README.md]: Heading like `Scaffolding` or `Creating Addons` should work better @fokusferit

File: README.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 		<img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
 	</a>
 </div>
+<p align="center">
+  A scaffolding tool for webpack.
```
[the remark this answers is not in the document]
Comment (ematipico) [reply README.md]: `The sole and official CLI of webpack` ?

File: README.md [hunk 3]
```diff
@@ -16,25 +19,56 @@
...
+Webpack CLI is a CLI tool for providing a flexible set of commands for developers to increase speed when setting up a custom webpack project. As of webpack v4, webpack is not expecting a configuration file but in many cases, developers want to create a more custom webpack configuration based on their use-cases and needs. Exactly all of these cases with webpack CLI we are providing a set of tools so improve the setup of custom webpack configuration. 
 
-`webpack-cli migrate <config>`
+### How to install
 
-[Read more about migrating](MIGRATE.md)
+When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed!
 
-## Creating new webpack projects
+Otherwise `npm install --save-dev webpack-cli` will install it.
```
Comment (ematipico) [inline README.md]: We should insert also the `yarn` snippet

File: README.md [hunk 4]
```diff
@@ -16,25 +19,56 @@
...
+Otherwise `npm install --save-dev webpack-cli` will install it. 
 
-The `init` feature allows users to get started with webpack, fast. Through scaffolding, people can create their own configuration in order to faster initialize new projects for various of use cases.
+### Commands
 
-`webpack-cli init webpack-scaffold-<package>`
+In order to simplify many common tasks when using webpack in your projects, webpack CLI provides different commands according to their use-cases.
 
-[Read more about scaffolding](SCAFFOLDING.md)
+webpack CLI is organized as a [multi-package repository](https://github.com/lerna/lerna) and every command is developed in the `packages` Folder.
```
Comment (ematipico) [inline README.md]: `webpack` => `Webpack`

Comment (dhruvdutt) [reply README.md]: https://webpack.js.org/branding/#the-name

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dhruvdutt) [reply README.md]: "sole" doesn't sound good to me. "official" should be suffice.

File: README.md (hunk 4 above)
Comment (ematipico) [reply README.md]: So we gotta change at the top, it was with first capital letter

File: README.md [hunk 5]
```diff
@@ -16,25 +19,56 @@
...
+  - [`webpack-cli serve`](./packages/serve/README.md#webpack-cli-serve)
+  - [`webpack-cli update`](./packages/update/README.md#webpack-cli-update)
+* [webpack.config.js](https://webpack.js.org/concepts/configuration/)
+* [Contributing and Internal Documentation](#contributing-and-internal-documentation)
 
-## Migration from webpack v1 to v2
+## About
 
-The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate` also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
+Webpack CLI is a CLI tool for providing a flexible set of commands for developers to increase speed when setting up a custom webpack project. As of webpack v4, webpack is not expecting a configuration file but in many cases, developers want to create a more custom webpack configuration based on their use-cases and needs. Exactly all of these cases with webpack CLI we are providing a set of tools so improve the setup of custom webpack configuration.
```
Comment (ematipico) [inline README.md]: `Webpack` => `webpack`

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ev1stensberg) [reply README.md]: Sole purpose is to compile configurations and build/bundle webpack, so I wouldn’t use that statement

File: README.md (hunk 1 above)
Comment (ev1stensberg) [reply README.md]: Getting started is better

File: README.md (hunk 5 above)
Comment (ev1stensberg) [reply README.md]: If it’s start of paragraph, capital, if not, underscore

File: README.md (hunk 1 above)
Comment (rishabh3112) [reply README.md]: Are you at line 65 ? @ev1stensberg

File: README.md (hunk 5 above)
Comment (ematipico) [reply README.md]: As @dhruvdutt pointed out before

https://webpack.js.org/branding/

> webpack should always be written in lower-case letters, even at the beginning of a sentence.

Comment (ev1stensberg) [reply README.md]: I disagree but okay..

**Comment by fokusferit** [comment]
- Thanks for reviewing this  🎊
- I will update this PR based on the feedback, so we can have a nice looking Readme 😄

**Pull request merged**